custom_components/powercalc/data/sonos/sub/model.json

```diff
@@ -0,0 +1,26 @@
+{
+    "measure_description": "Streamed Underworld - Dark & Long for this test",
+    "measure_method": "manual",
+    "measure_device": "Shelly Plug S",
+    "name": "Sonos Sub (Gen1)",
+    "standby_power": 5.23,
```

Comment:
That's some hefty standby power for these sonos devices. Good to know ;-)
That's what powercalc is about to find out about your usage.
Would be a good idea to put these behind a smart plug and shut down the power completely during night and when away.

Reply:
That is exactly what I do :) When nobody is home or when the bedtime automation runs, the sonos speakers go off :)